destinations/data/victoriametrics.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: internal.odigos.io/v1beta1
+kind: Destination
+metadata:
+  type: victoriametrics
```

Comment:
Since there is also OSS victoria metrics, perhaps we should call this one "victoriametricscloud" so that we can add the self-hosted destination later and not have collision?